### DISCUSSION TITLE
Size (in meters or meters-squared) of one tile?

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#46267] [Q&A] [answered]
As the subject line says - trying to figure out the thickness of a wall.

**Comment by linonetwo** [comment]
I think it is 1~2m, because when a vehicle move in 6km/h = 1.666 m/s , it move 1 tile per second.

**Reply by actual-nh** [reply]
Thanks!

**Comment by anothersimulacrum** [comment, marked as answer]
It is entirely context dependent, and there's no set size. For mapgen at least it can be reasonable to assume `1x1x4m`.

**Reply by actual-nh** [reply]
Thanks!
